ntbs-service/Pages/PreviousHistoryDetails/Edit.cshtml

```diff
@@ -0,0 +1,64 @@
+@page "{handler?}"
+@model ntbs_service.Pages_PatientTBHistory.EditModel
+@using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
+
+@{
+    ViewData["Title"] = "Notification - Patient TB History";
+}
+@{string errorClassName = "nhsuk-form-group--error";}
+<div><a asp-page="/Patients/Index">Back to List</a></div>
+
+<h2>Previous History Details for New Patient</h2>
+
+<div class="row">
+    <date-comparison inline-template>
+        <div class="col-md-5">
+            <form method="post">
+                <input type="hidden" asp-for="NotificationId" />
+                <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+
+                <div class="nhsuk-form-group">

```

Comment:
We can use the taghelpers for all of these elements though - see Personal Details page as an example :)